frontend/src/components/PartnerList.vue

```diff
@@ -175,35 +177,38 @@ export default {
       });
     },
 
-    fetch() {
-      this.$store.dispatch("loadPartnerSelectList");
-
-      partnerService
-        .getPartners(this.onlyIssuersForSchema)
-        .then((result) => {
-          console.log("Partner List", result);
-          if (Object.prototype.hasOwnProperty.call(result, "data")) {
-            this.isBusy = false;
-
-            if (this.onlyAries) {
-              result.data = result.data.filter((item) => {
-                return item.ariesSupport === true;
-              });
-            }
-
-            this.data = result.data.map((partner: PartnerAPI) => {
-              const tempPartner: PartnerAPI & { address: string } = {
-                address: this.getProfileAddress(partner),
-                ...partner,
-              };
-              return tempPartner;
+    async fetch() {
+      this.isBusy = true;
+      // this.$store.dispatch("loadPartnerSelectList");
```

Comment:
store dispatch still needed if not remove comment

Reply:
this needs some discussion.